src/chunk.c

```diff
@@ -2728,10 +2729,15 @@ ts_chunk_exists_with_compression(int32 hypertable_id)
 	init_scan_by_hypertable_id(&iterator, hypertable_id);
 	ts_scanner_foreach(&iterator)
 	{
-		bool isnull =
+		bool isnull_dropped;
+		bool isnull_chunk_id =
```

Comment:
I don't think we need these changes if the compressed_chunk_id is marked correctly as null (when the chunk is dropped but catalog entry is preserved)

Reply:
I think excluding chunks marked as dropped is more important than the other change

Reply:
yes, makes it clearer